renderer/lib/location-history.js

```diff
@@ -12,14 +12,16 @@ LocationHistory.prototype.url = function () {
 }
 
 LocationHistory.prototype.current = function () {
-  return this._history[this._history.length - 1]
+  return this._current
 }
 
 LocationHistory.prototype.go = function (page, cb) {
   if (!cb) cb = noop
   if (this._pending) return cb(null)
 
-  console.log('go', page)
+  if (process.env.NODE_ENV !== 'test') {
```

Comment:
I added this because log littered console on `npm test`. I guess there are two ways to avoid that:
- Don't call log function in unit-testable classes (we can always do that outside)
- Create a log wrapper that only writes to log when not in test environment

More details [here](http://stackoverflow.com/questions/22709882/how-to-suppress-application-logging-messages-from-a-node-js-application-when-run).

Also, if for some reason we don't want tests, we can just cherry-pick `ab32d04` commit. It doesn't  include any test code.